reflex/reflex.py

```diff
@@ -228,6 +240,8 @@ def export(
     console.rule("[bold]Compiling production app and preparing for export.")
 
     if frontend:
+        # never migrate the database during export
```

Comment:
same nit: Can we start all the comments with capital letters and end with periods everywhere